scripts/globals/mobskills/boiling_blood.lua

```diff
@@ -16,9 +16,8 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    xi.mobskills.mobBuffMove(mob, xi.effect.HASTE, 2500, 0, 60)
-    xi.mobskills.mobBuffMove(mob, xi.effect.BERSERK, 5000, 0, 60)
-    skill:setMsg(xi.msg.basic.NONE)
+    xi.mobskills.mobBuffMove(mob, xi.effect.HASTE, 25, 0, 60)
```

Comment:
this is incorrect. Hast is not scaled that way in the project at present. And you've lost the line break. You also don't clearly state what you are fixing in your commit message https://github.com/LandSandBoat/server/pull/3736/commits/0dbe6eb7bf49ec5d0a38c25d097103347a94285e or in your pr title. I presume the problem was gaining a MASSIVE amount of attack.

Reply:
looks like based on the script it would've added 5000%~ attack

Reply:
yup. but the hast is 25% when using 2500 of the modifier. and if mobBuffMove does math on its value ima kick someone in the shins. Hard.

Reply:
at min need the line break put back and commit message amended regardless.

Reply:
I rewrote the merge request and commit text. line break returned!

Reply:
https://github.com/LandSandBoat/server/blob/a5540475df64b3f70231ace3190c78f5130c0f4f/src/map/modifier.h#L227